src/main/java/homework_1/Main.java

```diff
@@ -1,9 +1,14 @@
 package homework_1;
 
 public class Main {
-
-  public static void main(String[] args) {
-    System.out.println("Hello homework!");
-  }
-
+    public static void main(String[] args) {
+        for (String arg : args) {
+            if (!arg.equals("error")){
+                System.out.println(arg + ": " + arg.length() + " letters");
+            } else{
+                System.err.println("Alarm!");
```

Comment:
Doesn't work as expected 
![image](https://user-images.githubusercontent.com/58872509/125207235-cea63100-e293-11eb-9dcc-ef14d9a21f07.png)

Reply:
@NikolaevArtem Could you provide some explanation? 
Technically I can't see which requirement hadn't met, but I can quess, that it's not recommended to use different streams in current task (out and err) because of undefined output order.